fbpic/boundaries/boundary_communicator.py

```diff
@@ -634,6 +723,26 @@ def exchange_domains( self, send_left, send_right, recv_left, recv_right ):
             req_rr.Wait()
             req_sr.Wait()
 
+    def mpi_buffer(self, gpu_array):
```

Comment:
Would you mind changing the name of this function to something else (e.g. `get_as_gpudirect_buffer`), so as to avoid confusions with the return variable for this function (`mpi_buffer`) as well as the object `mpi_buffers`.